.github/workflows/ecosystem-ci-trigger.yml

```diff
@@ -22,44 +46,123 @@ jobs:
                 repo: context.repo.repo,
                 username: user,
               });
-              hasTriagePermission = data.user.permissions.triage
+              hasTriagePermission = ['triage', 'write', 'admin'].includes(data.permission);
             } catch (e) {
               console.warn(e)
             }
 
+            const prData = ${{ steps.get-pr-data.outputs.result }}
+
+            const url = `https://pkg.pr.new/vite@${prData.commit}`
+            const response = await fetch(url)
```

Comment:
same as in ecosystem-ci repo, this just needs a head request to see if the resource is there, not fetch it